### PR TITLE
web/admin: Carrier/Company balances changed to ghost fields

### LINKED ISSUE #655: web/admin: ask balances to CGRateS live
In sections:

- Carriers
- Prepaid balances

Column balance is retrieved from database (Companies.balance and Carriers.balance) instead of asking to CGRateS as it is done with ASR/ACD values. Database value is updated in a timer.

This issue aims to:

- Ask CGRateS in realtime.
- Keep ivozprovider-balances timer.

### PR DESCRIPTION
**Pull request by Kaian**
This PR fixes #655 by replacing current balance fields with a ghost field that request current balance status to CGRateS.

The request already existed in a service so I just changed the field type and updated the existing ghosts.

### CODE REVIEW

File: web/admin/application/library/IvozProvider/Klear/Ghost/Carriers.php [hunk 1]
```diff
@@ -9,12 +11,20 @@ class IvozProvider_Klear_Ghost_Carriers extends KlearMatrix_Model_Field_Ghost_Ab
...
+    private $fetchCarrierBalance;
+
     public function __construct()
     {
         $serviceContainer = \Zend_Registry::get('container');
         $this->fetchCallStats = $serviceContainer->get(
             FetchCallStatsService::class
         );
+        $this->fetchCarrierBalance = $serviceContainer->get(
+            CarrierBalanceService::class
```
Comment (mmadariaga) [inline web/admin/application/library/IvozProvider/Klear/Ghost/Carriers.php]: We should use CompanyBalanceServiceInterface here.
You may need to set it public at library/CoreBundle/Resources/config/services/infraestructure/cgrates.yml

File: web/admin/application/library/IvozProvider/Klear/Ghost/Companies.php [hunk 2]
```diff
@@ -9,6 +10,19 @@
...
+    /**
+     * @var CompanyBalanceService
+     */
+    private $fetchCompanyBalance;
+
+    public function __construct()
+    {
+        $serviceContainer = \Zend_Registry::get('container');
+        $this->fetchCompanyBalance = $serviceContainer->get(
+            CompanyBalanceService::class
```
Comment (mmadariaga) [inline web/admin/application/library/IvozProvider/Klear/Ghost/Companies.php]: We should use CompanyBalanceServiceInterface here as well

File: web/admin/application/library/IvozProvider/Klear/Ghost/Carriers.php (hunk 1 above)
Comment (Kaian) [reply web/admin/application/library/IvozProvider/Klear/Ghost/Carriers.php]: Sure! Shouldn't the above service (FetchCallStats) also use an interface?

Comment (mmadariaga) [reply web/admin/application/library/IvozProvider/Klear/Ghost/Carriers.php]: Yes :-)

**Pull request merged**